WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/LargeValueFormatter.kt

```diff
@@ -0,0 +1,46 @@
+package org.wordpress.android.ui.stats.refresh.utils
+
+import com.github.mikephil.charting.components.AxisBase
+import com.github.mikephil.charting.data.Entry
+import com.github.mikephil.charting.formatter.IAxisValueFormatter
+import com.github.mikephil.charting.formatter.IValueFormatter
+import com.github.mikephil.charting.utils.ViewPortHandler
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Locale
+import kotlin.math.round
+
+class LargeValueFormatter : IValueFormatter, IAxisValueFormatter {
```

Comment:
Should we add a comment saying where the original implementation of this comes from? And possibly one to explain why the format's locale is overridden?